### PR TITLE
fix(CommunityEdit): content width should have fixed size

### LINKED ISSUE #9051: Edit Community Page - Content should be a fixed width and not resize when window is resized
### Description

It was agreed with the design team that, for consistency, the `Edit Community` page will maintain a fixed width as shown in the design and will not resize when the window is resized:

[Design](https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=3132%3A383870&t=Q7sGGTFUmcOTI1oq-0):

Screenshot 2023-01-11 at 13 04 28

Here the current implementation:

https://user-images.githubusercontent.com/97019400/211801970-5888902b-3b14-4e97-b658-f9258376f9a5.mov

### PR DESCRIPTION
**Pull request by alexandraB99**
Closes #9051

### What does the PR do
Fixes Community edit panel should have fixed width

### Affected areas
Community edit panel

### Screenshot of functionality (including design for comparison)

https://user-images.githubusercontent.com/31625338/212327821-0769e9af-753e-4d2f-8173-a7dd694a4619.mov

### CODE REVIEW

**Review by caybro — APPROVED** [review]
LGTM

File: ui/app/AppLayouts/Chat/panels/communities/CommunityEditSettingsPanel.qml [hunk 1]
```diff
@@ -46,13 +49,12 @@ StatusScrollView {
...
     readonly property bool saveChangesButtonEnabled: true
 
-    implicitWidth: layout.implicitWidth
-    implicitHeight: layout.implicitHeight
-
     ColumnLayout {
         id: layout
 
-        width: root.availableWidth
+        width: 640
```
Comment (noeliaSD) [inline ui/app/AppLayouts/Chat/panels/communities/CommunityEditSettingsPanel.qml:55]: Should it be 608px following the design?

**Review by noeliaSD — APPROVED** [review]
LGTM! Also tested!

Just a small comment added!

**Review by micieslak — CHANGES_REQUESTED** [review]
Looks ok, one minor doubt here.

File: ui/app/AppLayouts/Chat/controls/community/CommunityBannerPicker.qml [hunk 2]
```diff
@@ -23,8 +23,8 @@ Item {
     property alias cropRect: editor.cropRect
     property string imageData
 
+    implicitWidth: parent.width
```
[the remark this answers is not in the document]
Comment (micieslak) [reply ui/app/AppLayouts/Chat/controls/community/CommunityBannerPicker.qml:26]: From what I see `CommunityBannerPicker` is used in layout with `Layout.fillWidth: true` so this implicit width will be ignored in this case.

**Comment by alexandraB99** [comment]
> LGTM! Also tested!
> 
> Just a small comment added!

correct, I overlooked at the margins, fixed!

File: ui/app/AppLayouts/Chat/controls/community/CommunityBannerPicker.qml (hunk 2 above)
[the remark this answers is not in the document]
Comment (alexandraB99) [reply ui/app/AppLayouts/Chat/controls/community/CommunityBannerPicker.qml]: removed!

**Review by micieslak — APPROVED** [review]
lgtm!

**Pull request merged**